_maps/map_files/NorthStar/north_star.dmm

```diff
@@ -31413,7 +31413,10 @@
 /area/station/command/heads_quarters/cmo)
 "ihg" = (
 /obj/effect/decal/cleanable/dirt,
-/obj/item/toy/plush/lizard_plushie/green,
+/obj/item/toy/plush/lizard_plushie/green{
+	name = "Bites-The-Wires";
```

Comment:
no map edits, no player refereces

Reply:
> no map edits, no player refereces

Oh is that someone's static? I was just picking a generic Argonian name that would work for a maint lizard.
I can change that no sweat.

Reply:
oh, in that case just make these actual plushie subtypes in the code and place them we dont do var edited items on maps

Reply:
Non-reused var edits are completely fine

Reply:
>  we dont do var edited items on maps

tell that to every edited plushie on every map